Document/0x06a-Platform-Overview.md

```diff
@@ -167,3 +167,17 @@ The following APIs [require user permission](https://www.apple.com/business/docs
 - Bluetooth sharing
 - Media Library
 - Social media accounts
+
+### DeviceCheck
+
+The DeviceCheck framework, including its components DeviceCheck and App Attest, helps you prevent fraud in promotional offers. It allows you to persistently store information on the device and on Apple servers to limit promotions to once per device. The stored information remains intact across app reinstallation, device transfers, and even resets. You can reset this data periodically.
+
+For more information, refer to the [DeviceCheck documentation](https://developer.apple.com/documentation/devicecheck "DeviceCheck documentation").
+
+### App Attest

```

Comment:
Should we keep it within DeviCheck since it's part of it?

```suggestion
#### App Attest
```

Reply:
Yes, that definitely sounds like a better idea.